source/patterns/@aws-solutions-constructs/aws-lambda-sagemakerendpoint/README.md

```diff
@@ -0,0 +1,132 @@
+
+## Pattern Properties
+
+| **Name**                 | **Type**                                                                                                                       | **Description**                                                                                                                 |
+| :----------------------- | :----------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| lambdaFunction           | [`lambda.Function`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.Function.html)                         | Returns an instance of the Lambda function created by the pattern.                                                              |
+| sageMakerEndpoint        | [`sagemaker.CfnEndpoint`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-sagemaker.CfnEndpoint.html)             | Returns an instance of the SageMaker Endpoint created by the pattern.                                                           |
+| sageMakerEndpointConfig? | [`sagemaker.CfnEndpointConfig`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-sagemaker.CfnEndpointConfig.html) | Returns an instance of the SageMaker EndpointConfig created by the pattern, if `existingSageMakerEndpointObj?` is not provided. |
+| sageMakerModel?          | [`sagemaker.CfnModel`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-sagemaker.CfnModel.html)                   | Returns an instance of the SageMaker Model created by the pattern, if `existingSageMakerEndpointObj?` is not provided.          |
+| vpc?                     | `ec2.IVpc`                                                                                                                     | Returns an instance of the VPC created by the pattern, if `deployVpc?` is `true`, or `existingVpc?` is provided.                |
+
+## Default settings
+
+Out of the box implementation of the Construct without any override will set the following defaults:
+
+### AWS Lambda Function
+
+- Configure limited privilege access IAM role for Lambda function.
+- Enable reusing connections with Keep-Alive for NodeJs Lambda function.
+- Allow the function to invoke the SageMaker endpoint for Inferences.
+- Configure the function to access resources in the VPC, where the SageMaker endpoint is deployed.
+- Enable X-Ray Tracing.
```

Comment:
Document the environment variable defined (we just added this to the docs for other constructs - it should be online by the tine you read this.)

Also - add SAGEMAKER to the variable name, don't leave it as just ENDPOINT (it's ambiguous and doesn't match the others which start with service name, eg SQS_QUEUE_URL)

Reply:
Will do.

Reply:
I don't see the environment variable documented on the README page - am I missing it somewhere? Model after [this](https://docs.aws.amazon.com/solutions/latest/constructs/aws-lambda-sqs.html):

## AWS Lambda Function
* Configure limited privilege access IAM role for Lambda function.
* Enable reusing connections with Keep-Alive for NodeJs Lambda function.
* Allow the function to only send messages to the queue (purging can be enabled using the enableQueuePurge property).
* Enable X-Ray tracing
* Set environment variables:
* * SQS_QUEUE_URL
* * AWS_NODEJS_CONNECTION_REUSE_ENABLED (for Node 10.x and higher functions)